### PR TITLE
make alert metrics count type & add service heartbeat

### PR DESCRIPTION
**Pull request by kunmingg**
1. make alert metrics count type
2. add service heartbeat

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/1849)

### CODE REVIEW

File: bootstrap/cmd/bootstrap/app/ksServer.go [hunk 1]
```diff
@@ -1137,5 +1165,6 @@ func (s *ksServer) StartHttp(port int) {
 	// add an http handler for prometheus metrics
 	http.Handle("/metrics", promhttp.Handler())
 
+	go countHeartbeat()
```
Comment (r2d4) [inline bootstrap/cmd/bootstrap/app/ksServer.go]: nit: you could probably inline this one as

```go
go func() {
   time.Sleep(10 * time.Second)
   serviceHeartbeat.Inc()
}()
```

Comment (abhi-g) [reply bootstrap/cmd/bootstrap/app/ksServer.go]: Sorry, added the labels too soon...

Comment (r2d4) [reply bootstrap/cmd/bootstrap/app/ksServer.go]: It works the same either way :)

**Pull request merged**